src/local_scheduler/format/local_scheduler.fbs

```diff
@@ -84,9 +84,11 @@ table RegisterClientRequest {
 table DisconnectClient {
 }
 
-table ReconstructObject {
-  // Object ID of the object that needs to be reconstructed.
-  object_id: string;
+table ReconstructObjects {
```

Comment:
Seems like the method should really be called `fetch` instead of `reconstruct` and instead of taking a bool for `fetch` it should take a bool for `reconstruct`.

I'm ok with doing the name change later.